### PR TITLE
Creates the Logz Docs

### PR DESCRIPTION
**Pull request by JeffNeff**
https://github.com/triggermesh/docs/issues/133

### CODE REVIEW

**Comment by sebgoa** [comment]
@JeffNeff were are we on this, is this still a draft ?

File: docs/targets/logz.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Prerequisites
+* Logz.io account.
+* A Logz.io Shipping [token](https://docs.logz.io/user-guide/tokens/)
+
+
+## Deploying an Instance of the Target
+Open the Bridge creation screen and add a Target of type `Logz`.
+
+![Adding a Logz Target](../images/logz-target/create-bridge-1.png)
```
Comment (daceynolan) [inline docs/targets/logz.md:12]: This image is super big, is there any way we can change it?

File: docs/targets/logz.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Event Target for Logz.io 
+This event target integrates with Logz.io, using received CloudEvent's to ship messages to [Logz.io](https://logz.io/)
+
+## Prerequisites
+* Logz.io account.
```
Comment (daceynolan) [inline docs/targets/logz.md:5]: We do not need the ending period since this is not a complete sentence. 

What do you think of adding a link to the create account page for log.io?

File: docs/targets/logz.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* A Logz.io Shipping [token](https://docs.logz.io/user-guide/tokens/)
```
Comment (daceynolan) [inline docs/targets/logz.md:6]: We can remove "a" here to be consistent. Also I would lowercase "Shipping"

File: docs/targets/logz.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+
+## Deploying an Instance of the Target
+Open the Bridge creation screen and add a Target of type `Logz`.
+
+![Adding a Logz Target](../images/logz-target/create-bridge-1.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **Shipping Token** The API Shipping Token.
```
Comment (daceynolan) [inline docs/targets/logz.md:16]: Should we add more infor about what an API shipping token is. This seems redundant to have Shipping Token - Api Shipping token. I would either add what a shipping token is or just remove the redundancy

Comment (daceynolan) [inline docs/targets/logz.md:16]: I am not a fan of the word "the" on both of these bulleted points. Since these are not complete sentences I think it can be removed. I also would remove the ending periods since these are not complete sentences.

File: docs/targets/logz.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+## Deploying an Instance of the Target
+Open the Bridge creation screen and add a Target of type `Logz`.
+
+![Adding a Logz Target](../images/logz-target/create-bridge-1.png)
+
+In the Target creation form, provide a name for the event Target, and add the following information:
+
+* **Shipping Token** The API Shipping Token. 
+* **Logs Listener URL** The API endpoint. (Can be found above your Shipping token in the Logz.io dashboard)
```
Comment (daceynolan) [inline docs/targets/logz.md:17]: Here I would rephrase this as "An API endpoint that can be found above your shipping token in the Logz.io dashboard. Also, shipping does not need to be capitalized here.

File: docs/targets/logz.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+![Adding a Target](../images/logz-target/create-bridge-2.png)
```
Comment (daceynolan) [inline docs/targets/logz.md:19]: This image is really big too.

File: docs/targets/logz.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+
+Click the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
```
Comment (daceynolan) [inline docs/targets/logz.md:22]: Maybe we should say "After clicking the `Save` button..."

File: docs/targets/logz.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with was successfully created.
+
+## Event Types
+
```
Comment (daceynolan) [inline docs/targets/logz.md:27]: Do we need to add some sentence like this? "A Logz event Target accepts the following CloudEvent types:"

**Review by daceynolan — CHANGES_REQUESTED** [review]
Sorry I have a lot of nitpicky comments. 

Also, be sure to add this new Target to the lists of Targets in the docs.

**Comment by daceynolan** [comment]
@JeffNeff could you create an issue for this and link the issue here please

**Comment by sebgoa** [comment]
Dacey is correct, it is better to create an issue and then reference it in a PR. That way we know when someone is tackling an issue because the person assigns the issue to themselves.

**Review by daceynolan — APPROVED** [review]
This looks good, please just add the issue to the top description so they can be linked and to keep track of records wise.
